.agents/skills/create-hurl-test/SKILL.md

```
@@ -0,0 +1,420 @@
+[Request body — for POST/PUT]
+```json
+{ "key": "value" }
+```
+
+HTTP status_code
+[Asserts]
+jsonpath "$.field" == "value"
+
+[Captures]
+var_name: jsonpath "$.id"
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)